app/backend/src/couchers/migrations/versions/74e7f371be76_make_contribute_ways_a_non_null_array.py

```diff
@@ -0,0 +1,23 @@
+"""Make contribute_ways a non-null array
+
+Revision ID: 74e7f371be76
+Revises: 725813da484d
+Create Date: 2021-09-06 11:31:50.349578
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "74e7f371be76"
+down_revision = "725813da484d"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("UPDATE contributor_forms SET contribute_ways = '{}' WHERE contribute_ways IS NULL")
```

Comment:
Where's the `'{}'` syntax documented? I can only see `ARRAY[]::string[]` to denote empty array described [here](https://www.postgresql.org/docs/13/sql-expressions.html#SQL-SYNTAX-ARRAY-CONSTRUCTORS)

Reply:
Yeah doesn't seem like it's documented there! I just looked at the database and saw that's how the empty arrays are stored, and it seems to work :P